### ISSUE TITLE
🚀 - Add class to overlay - like what was done for the ng-bootstrap version

### ISSUE THREAD

**Issue by kpaxton** [hakimio/ngx-ui-tour#121]
### Which ngx-ui-tour-* package is relevant to the feature request?

md-menu

### Description

Adding a class parameter to the step that get's appended to the overlay that is created will allow for additional styling of each step at a higher level..
This: https://github.com/hakimio/ngx-ui-tour/pull/113, just for md-menu

**Comment by hakimio** [comment]
It already has [`tour-step` class](https://github.com/hakimio/ngx-ui-tour/blob/d1401a049e74c3585ad1248329aef60654b11c59/projects/ngx-ui-tour-md-menu/src/lib/tour-step-template/tour-step-template.component.html#LL1C1-L1C1) applied to `mat-menu`. You can target it with `.mat-menu-panel.tour-step` or `mat-menu.tour-step`.

**Comment by kpaxton** [comment]
Yes, I do see that, and I am targeting that. However, I'd like to be able to style steps individually. So providing a place in the step configuration to provide additional classes for styling would be helpful.
